Make ECDSA_do_sign_ex() (which is pointless without ECDSA_sign_setup())
fall back to ECDSA_do_sign().

Index: ECDSA.xs
--- ECDSA.xs.orig
+++ ECDSA.xs
@@ -75,7 +75,7 @@ ECDSA_do_sign_ex(const unsigned char *dgst, const BIGN
                 STRLEN dgst_len;
 	CODE: 
                 dgst = (const unsigned char *)SvPV(ST(0), dgst_len);
-                RETVAL = ECDSA_do_sign_ex(dgst, dgst_len, kinv, rp, eckey);
+                RETVAL = ECDSA_do_sign(dgst, dgst_len, eckey);
 	OUTPUT:
 		RETVAL
 
